projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -2678,11 +2675,11 @@
 
         &::after {
             @extend %filtering-scroll-mask;
-            #{$left}: calc(100% + 6px);
-            background: linear-gradient(to #{$right}, var-get($theme, 'filtering-row-background'), transparent);
+            inset-inline-start: calc(100% + 6px);
+            background: linear-gradient(to end, var-get($theme, 'filtering-row-background'), transparent);
```

Comment:
You said this one doesn't work this way?

Reply:
Correct, this has been resolved in the latest commit.